http.gemspec

```diff
@@ -19,7 +19,9 @@ Gem::Specification.new do |gem|
   gem.licenses      = ["MIT"]
 
   gem.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
-  gem.files         = `git ls-files`.split("\n")
+  gem.files         = Dir["lib/**/*.rb"] +
+                      %w[http.gemspec README.md CHANGES.md LICENSE.txt]
```

Comment:
IMO it's better to keep git ls-files + filter instead. And we can update git ls-files command a bit:

``` ruby
  gem.files         = Dir.chdir(__dir__) do
    lib_files = `git ls-files -z`.split("\x0").
      select { |f| f.start_with?("lib/") }

    lib_files + %w[http.gemspec README.md CHANGES.md LICENSE.txt]
  end
```